DocuSignSamplePlugin/DocuSignPlugin/src/main/java/com/ibm/icn/extension/docusign/service/DocuSignConfiguration.java

```diff
@@ -14,15 +14,14 @@
 /*****************************************************************************************************************
  * This class stores the DocuSign credentials required to connect to DocuSign systems.
  * The credentials required to authenticate to a DocuSign system are 
- * 				userName
- * 				password
+ * //Deprecated.
+ * //TODO: Migrate TO OAUTH2.0
```

Comment:
Is this done or still a todo item?

Reply:
Hi Andy, Thanks for the catch, Making those changes now